src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewUnitAddressLinkedProcessor.java

```diff
@@ -58,7 +58,7 @@ public UUID process(OutcomeSuperSetDto outcome, UUID caseIdHolder, String type)
     root.put("sourceCase", "NEW_UNIT");
     root.put("outcome", outcome);
     root.put("newCaseId", caseId);
-    root.put("sourceCaseId", outcome.getSiteCaseId());
+    root.put("sourceCaseId", "9170085a-1367-4270-8674-acc6d72671e1");
```

Comment:
Hard coded case id?

Reply:
This has been updated. Not sure how this even got there!